.gitignore

```diff
@@ -5,3 +5,4 @@
 Getting-Started-in-R_files
 Getting-Started-in-R_cache
 local
+**/*.log
```

Comment:
Fancy. As there can be only one I usually just write it out but I tip my head...